cmd/nvidia/main.go

```diff
@@ -2,21 +2,60 @@ package main
 
 import (
 	"flag"
-
+	"fmt"
 	"github.com/AliyunContainerService/gpushare-device-plugin/pkg/gpu/nvidia"
+	"github.com/AliyunContainerService/gpushare-device-plugin/pkg/kubelet/client"
 	log "github.com/golang/glog"
+	"io/ioutil"
+	"k8s.io/client-go/rest"
+	"time"
 )
 
 var (
-	mps         = flag.Bool("mps", false, "Enable or Disable MPS")
-	healthCheck = flag.Bool("health-check", false, "Enable or disable Health check")
-	memoryUnit  = flag.String("memory-unit", "GiB", "Set memoryUnit of the GPU Memroy, support 'GiB' and 'MiB'")
+	mps            = flag.Bool("mps", false, "Enable or Disable MPS")
+	healthCheck    = flag.Bool("health-check", false, "Enable or disable Health check")
+	memoryUnit     = flag.String("memory-unit", "GiB", "Set memoryUnit of the GPU Memroy, support 'GiB' and 'MiB'")
+	kubeletAddress = flag.String("kubelet-address", "0.0.0.0", "Kubelet IP Address")
```

Comment:
Could you provide the flag to enable/disable the function to query from kubelet?

Reply:
> Could you provide the flag to enable/disable the function to query from kubelet?

done